### PR TITLE
Gha to build and deploy

### PR DESCRIPTION
**Pull request by dennys-bd**

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
+{
+    "name": "Cdrf Env",
+    // "dockerComposeFile": "docker-compose.yml",
+    // "service": "devcontainer",
```
Comment (fjsj) [inline .devcontainer/devcontainer.json]: Remove commented code if possible

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: pip install -r requirements.txt
+      - name: Build
+        run: fab build
+      - name: Check if worked
+        run: |
+          cd public
+          ls
```
Comment (fjsj) [inline .github/workflows/build.yml]: Does ls returns non-zero if empty dir?

**Review by fjsj — CHANGES_REQUESTED** [review]
Please check comments

File: .github/workflows/build.yml (hunk 2 above)
Comment (dennys-bd) [reply .github/workflows/build.yml]: I would check the logs to see what ls would print, but I'll be using a test bucket

**Review by fjsj — APPROVED** [review]

**Pull request merged**